university-agent/src/main/java/nl/quintor/studybits/service/AgentService.java

```diff
@@ -88,7 +88,7 @@ public MessageEnvelope login(String studentId) throws IndyException, ExecutionEx
 
         studentService.setConnectionData(studentId, connectionRequest.getDid(), connectionRequest.getRequestNonce());
 
-        return new MessageEnvelope<>(IndyMessageTypes.CONNECTION_REQUEST, connectionRequest, null, null, null);
+        return new MessageEnvelope<>(IndyMessageTypes.CONNECTION_REQUEST, connectionRequest, null, universityTrustAnchor, null);
```

Comment:
I think this universityTrustAnchor parameter is left out on purpose here, since the connectionRequest does not need to be encrypted.